atomsci/ddm/utils/struct_utils.py

```diff
@@ -381,9 +379,9 @@ def _merge_values(values, strategy='list'):
     elif strategy == 'min':
         val = min(values)
     else:
-        raise Exception('Unknown column merge strategy: %s', columnmerge)
+        raise Exception('Unknown column merge strategy: %s' % (strategy) )
 
-    if type(val) is list and len(val) == 1:
+    if isinstance(val, list) and len(val) == 1:
```

Comment:
test?

Reply:
This is only used in the deprecated function, _merge_dataframes_by_smiles. I think this should be removed soon.